src/Assets/styleController.ts

```diff
@@ -43,6 +50,13 @@ const themes: IThemes = {
       'font-size': '16px',
       '--main-header-color': '#B85A27',
       '--midBlue-color': '#41528C',
+      '--option-card-background-color': '#FFFFFF',
+      '--option-card-hover-background-color': '#ECDEED',
```

Comment:
Can you pull from the feature branch. On the feature branch, there is a variable called `hoverColor`. What do you think about calling this CSS variable `--hover-color` for consistency?